tests/reader_test.cpp

```diff
@@ -148,6 +168,8 @@ TEST(ReaderTest, syntax_error)
     "   (myfloat 1.125 err)\n\r"
     "   (mystring \"Hello World\" err)\n"
     "   (mystringtrans (_ \"Hello World\" err))\n"
+    "   (mycompressedintarray1 -5 0 43 -5 -12 3 -1 5)\n"
```

Comment:
Could we also test something like -5 -5 -5?

Also im curious what happens with -0 but im sure sexpr turns it to 0.

Reply:
"-5 -5 -5" would break on the 2nd integer.

Reply:
oh my goodness im friggin blind. Forgive me